spec/features/availability_spec.rb

```diff
@@ -65,84 +103,52 @@
-        expect(page).to have_selector '.library-location', text: 'ReCAP'
+        visit '/catalog/857469'
+        expect(page).to have_selector '.availability--online a', minimum: 1
       end
-    end
-  end
 
-  # There are issues with correctly setting the user agent with chromedriver
-  context 'when visited by an indexing bot', driver: :iphone do
-    before do
-      page.driver.add_headers('User-Agent' => 'Mozilla/5.0 (iPhone; CPU iPhone OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143 Safari/601.1 (compatible; AdsBot-Google-Mobile; +http://www.google.com/mobile/adsbot.html)')
+      it 'display umlaut links for marcit record within the online section', unless: in_ci? do
+        visit '/catalog/9774256'
+        expect(page).to have_selector '.availability--online .umlaut .fulltext', count: 1
+        expect(page).to have_selector '.availability--online .umlaut .fulltext .response_item', minimum: 4
+      end
     end
 
-    describe 'viewing a catalog record', js: true do
-      before do
+    describe 'Multiple items all available', js: true do
+      it 'display availability as on-site and does not display individual items', unless: in_ci? do
         stub_holding_locations
-        visit '/catalog/3256177'
-      end
-
-      xit 'does not render the holding section' do
-        expect(page).not_to have_selector('.availability--holding')
+        visit 'catalog/857469'
+        expect(page).to have_selector '.availability-icon.badge.badge-success', text: 'All items available'
       end
     end
 
-    describe 'viewing a record for an electronic holding', js: true do
-      before do
+    describe 'On-site multiple items all available', js: true do
+      it 'displays availability as on-site and does not display individual items', unless: in_ci? do
         stub_holding_locations
-        visit '/catalog/857469'
-      end
-
-      xit 'does not render the online section' do
-        expect(page).not_to have_selector('.availability--online')
+        visit 'catalog/7777379'
+        expect(page).to have_selector '.availability-icon.badge.badge-success', text: 'On-site access', count: 1
       end
     end
 
-    describe 'viewing a record for a holding in a temp location', js: true do
-      before do
+    # these are now available
+    xdescribe 'On-site multiple items unavailable', js: true do
+      it 'displays See front desk and does not display individual items', unless: in_ci? do
         stub_holding_locations
-        visit '/catalog?q=7917192'
+        visit 'catalog/2238036'
+        expect(page).to have_selector '.availability-icon.badge.badge-success', text: 'See front desk', count: 1
       end
+    end
 
-      xit 'does not render the location' do
-        expect(page).not_to have_selector('.availability--holding')
+    # the item used here is no longer checked out
+    xdescribe 'Checked out item', js: true do
+      it 'shows due date', unless: in_ci? do
```

Comment:
can you update this with an item that is checked out?

Reply:
Same comment as above. We're skipping these because they hit live data that keeps changing. Also, IIRC, we will not be able to get access to checked-out status in Alma. The availability will simply be "Unavailable", so this is a "voyager" spec that will not migrate.